api/logs/views.py

```diff
@@ -29,8 +32,24 @@ def get_log(self):
             )
         return log
 
+    def check_log_permission(self, log):
+        """
+        Cycles through nodes on log backrefs.  If user can view any of the nodes, the log can be viewed.
+        """
+        auth_user = get_user_auth(self.request)
+        log_nodes = []
+
+        for node_id in log._backrefs['logged']['node']['logs']:
+            node = get_object_or_error(Node, node_id, display_name='node')
```

Comment:
But as you [hint at](https://github.com/CenterForOpenScience/osf.io/pull/4996/files#r52082595), `get_object_or_error` [doesn't error](https://github.com/CenterForOpenScience/osf.io/blob/ea8186dcff172b117b4ebe6632479d532f2ae77a/api/base/utils.py#L55-L86) with errors relevant to checking log permissions.